ansible/playbook.yml

```diff
@@ -31,20 +31,14 @@
         src: files/config
         dest: "{{ ansible_env.HOME }}/.aws/config"
 
-    - name: Copy dandi plugin
-      copy:
-        src: girder-dandi.tar.gz
-        dest: /tmp/girder-dandi.tar.gz
-        force: true
-
     - name: Install Girder plugins
       pip:
         name:
           - girder-oauth
           - girder-sentry
           - girder-hashsum-download
           - girder-download-statistics
-          - /tmp/girder-dandi.tar.gz
+          - "git+http://github.com/dandi/dandiarchive@{{ lookup('env', 'CIRCLE_SHA1') }}#subdirectory=girder-dandi-archive"
```

Comment:
should this be https (I assume github upgrades insecure requests anyway)?

Reply:
Yes, you are right